apps/web-giddh/src/app/shared/header/components/hamburger-menu/hamburger-menu.component.ts

```diff
@@ -38,6 +40,11 @@ export class HamburgerMenuComponent implements OnInit, OnDestroy {
         this.store.pipe(select(state => state.session.commonLocaleData), take(1)).subscribe((response) => {
             this.commonLocaleData = response;
         });
+        this.breakPointObservar.observe([
+            '(max-width: 767px)'
+        ]).pipe(takeUntil(this.destroyed$)).subscribe(result => {
+            this.isMobileView = result?.breakpoints['(max-width: 767px)'];
```

Comment:
Since single break point of `(max-width: 767px)` is used therefore `matches` key is enough. Please check it after changing.
```suggestion
            this.isMobileView = result?.matches;
```

Reply:
done